package-scripts.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     },
     test: {
       default: 'cross-env NODE_ENV=test jest --coverage',
+      update: 'cross-env NODE_ENV=test jest -u',
```

Comment:
This is why I prefer direct npm scripts. You can pass flags easier: `npm test -- -u`. I don't know how to pass flags with `nps`